### PR TITLE
Putting button

### PR DESCRIPTION
**Pull request by Aditi-0725**
Creating a button on 'Create New Auction' instead of just a heading.

### CODE REVIEW

File: pages/auction/index.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ const Auctions = () => {
       <NavBar personData={personData} />
       <Link href="/auction/new">
         <a>
-          <h2 className={styles.auctionPageNavigation}>Create New Auction</h2>
+          <h2 className={styles.auctionPageNavigation}>
+            {' '}
+            <button>Create New Auction</button>
+          </h2>
```
Comment (swarajpure) [inline pages/auction/index.js:22]: We wish to get rid of the h2 😅

**Review by swarajpure — CHANGES_REQUESTED** [review]
Why is package-lock changed? Also there's an unnecessary folder. Please delete it

**Comment by MehulKChaudhari** [comment]
@Aditi-0725 Are you working on this feature?

File: pages/auction/index.js (hunk 1 above)
Comment (Neha) [reply pages/auction/index.js:22]: Any reason why` h2` and `button` is wrapped with `a` tag? 

`a` and `button` is not required here as `Link` is handling the redirection.

**Comment by rajat-mehra05** [comment]
I'd like to pick up this issue. Please assign it to me.

**Comment by MehulKChaudhari** [comment]
> I'd like to pick up this issue. Please assign it to me.

Assigned, please reach out if you need any help.

**Comment by gurleen3** [comment]
Hey! @rajat-mehra05 I would like to take up this issue.

**Comment by rajat-mehra05** [comment]
> > I'd like to pick up this issue. Please assign it to me.
> 
> Assigned, please reach out if you need any help.

Hey @MehulKChaudhari, I have assigned the issue to @gurleen3. Can you please transfer the PR review to @gurleen3 ? Thanks.

**Comment by MehulKChaudhari** [comment]
> > > I'd like to pick up this issue. Please assign it to me.
> > 
> > 
> > Assigned, please reach out if you need any help.
> 
> Hey @MehulKChaudhari, I have assigned the issue to @gurleen3. Can you please transfer the PR review to @gurleen3 ? Thanks.

Sure

**Comment by MehulKChaudhari** [comment]
Closing this PR as we already have this button in master branch and it don't need any change as of now.

**Pull request closed without merging**